packages/react-router/tests/fileRoute.test-d.tsx

```diff
@@ -0,0 +1,87 @@
+import { expectTypeOf, test } from 'vitest'
+import { createFileRoute, createRootRoute } from '../src'
+
+const rootRoute = createRootRoute()
+
+const indexRoute = createFileRoute('/')()
+
+const invoicesRoute = createFileRoute('/invoices')()
+
+const invoiceRoute = createFileRoute('/invoices/$invoiceId')()
+
+const postLayoutRoute = createFileRoute('/_postLayout')()
+
+const postsRoute = createFileRoute('/_postLayout/posts')()
+
+const postRoute = createFileRoute('/_postLayout/posts/$postId_')()
+
+const authRoute = createFileRoute('/(auth)')()
```

Comment:
is this even possible? I thought a route group was merely a way to prevent a folder to show up in the url, but it cannot hold routes itself.

Reply:
Probably not. I will make it just the child